lib/Swagger/src/SiteAttribute/Api/SiteAttributeInternalApi.php

```diff
@@ -0,0 +1,217 @@
+     * @return array of \Swagger\Client\SiteAttribute\Models\SiteAttributeHalResponse, HTTP status code, HTTP response headers (array of strings)
+     * @throws \Swagger\Client\ApiException on non-2xx response
+     */
+    public function internallySaveAttributeWithHttpInfo($site_id, $attr_name, $x_wikia_schwartz = null, $body = null)
+    {
+        // verify the required parameter 'site_id' is set
+        if ($site_id === null) {
+            throw new \InvalidArgumentException('Missing the required parameter $site_id when calling internallySaveAttribute');
+        }
+        // verify the required parameter 'attr_name' is set
+        if ($attr_name === null) {
+            throw new \InvalidArgumentException('Missing the required parameter $attr_name when calling internallySaveAttribute');
+        }
+        // parse inputs
+        $resourcePath = "/internal/site/{siteId}/attr/{attrName}";
+        $httpBody = '';
+        $queryParams = array();
+        $headerParams = array();
+        $formParams = array();
+        $_header_accept = $this->apiClient->selectHeaderAccept(array('application/hal+json'));
+        if (!is_null($_header_accept)) {
+            $headerParams['Accept'] = $_header_accept;
+        }
+        $headerParams['Content-Type'] = $this->apiClient->selectHeaderContentType(array());
+
+        // header params
+        if ($x_wikia_schwartz !== null) {
+            $headerParams['X-Wikia-Schwartz'] = $this->apiClient->getSerializer()->toHeaderValue($x_wikia_schwartz);
+        }
+        // path params
+        if ($site_id !== null) {
+            $resourcePath = str_replace(
+                "{" . "siteId" . "}",
+                $this->apiClient->getSerializer()->toPathValue($site_id),
+                $resourcePath
+            );
+        }
+        // path params
+        if ($attr_name !== null) {
+            $resourcePath = str_replace(
+                "{" . "attrName" . "}",
+                $this->apiClient->getSerializer()->toPathValue($attr_name),
+                $resourcePath
+            );
+        }
+        // default format to json
+        $resourcePath = str_replace("{format}", "json", $resourcePath);
+
+        // body params
+        $_tempBody = null;
+        if (isset($body)) {
+            $_tempBody = $body;
+        }
+
+        // for model (json/xml)
+        if (isset($_tempBody)) {
+            $httpBody = $_tempBody; // $_tempBody is the method argument, if present
+        } elseif (count($formParams) > 0) {
+            $httpBody = $formParams; // for HTTP post (form)
+        }
+        // make the API Call
+        try {
+            list($response, $statusCode, $httpHeader) = $this->apiClient->callApi(
+                $resourcePath,
+                'PUT',
+                $queryParams,
+                $httpBody,
+                $headerParams,
+                '\Swagger\Client\SiteAttribute\Models\SiteAttributeHalResponse',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "\Swagger\Client\SiteAttribute\Models\SiteAttributeHalResponse" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)